aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -443,6 +443,125 @@ void THTensor_(indexFill)(THTensor *tensor, int dim, THLongTensor *index, scalar
   THLongTensor_free(index);
 }
 
+void THTensor_(gather)(THTensor *tensor, THTensor *src, int dim, THLongTensor *index)
```

Comment:
Worth mentioning that is pure move, no line changes introduced (at least `git diff --color-moved` thinks so)

Reply:
I added description that its mostly moving stuff around, but couldn't say that its only that as i have added attomicAdd overload.